gateway/ln-gateway/src/utils.rs

```diff
@@ -35,3 +42,11 @@ where
         }
     }
 }
+
+/// Hash FederationId to u64
+pub fn hash_federation_id(federation_id: FederationId) -> u64 {
+    let mut hasher = DefaultHasher::new();
```

Comment:
Since this isn't a cryptographically secure hash function an attacker could choose a federation name to cause a collision and potentially overwrite config. I'd advise to go with sha256 and hex encoding.